Cargo.toml

```diff
@@ -276,8 +275,8 @@ yamux = { opt-level = 3 }
 zeroize = { opt-level = 3 }
 
 [profile.release]
-# Substrate runtime requires unwinding.
-panic = "unwind"
+panic = "abort"
```

Comment:
@pepyakin Do we need `panic = "unwind"` for wasmtime?
Otherwise, with no native runtime, we can switch to `panic = "abort"` and gain better performance and significantly reduced binary size.

Reply:
I assume you mean whether traps generated by wasm code in wasmtime require `unwind`, and the answer is no, it works via a setjmp/longjmp like approach.

However, I think we should do this as a follow-up. The reason is, this requires some research to proof that we do not rely on this property somewhere else, because it was already there. I personally cannot recall any but it's better be safe than sorry.

Reply:
https://github.com/paritytech/substrate/issues/10874